index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<meta charset="utf8" />
+<html>
+  <script type="text/javascript" src="data.js"></script>
```

Comment:
likewise, failure-stats.json or so? Does it have to be a JS wrapper, or could this use XMLHttpRequest to load a .json document directly?

Reply:
It (probably) does not need to be, but I kind of like it more like that, than using HttpRequest...